CI/appveyor.install.ps1

```diff
@@ -238,9 +238,9 @@ function InstallLua() {
 }
 
 function InstallPcre() {
-  DownloadFile "https://ftp.pcre.org/pub/pcre/pcre-8.38.tar.gz" "pcre-8.38.tar.gz"
-  ExtractTar "pcre-8.38.tar.gz" "pcre-8.38"
-  Set-Location pcre-8.38\pcre-8.38
+  DownloadFile "https://ftp.pcre.org/pub/pcre/pcre-8.42.tar.gz" "pcre-8.42.tar.gz"
+  ExtractTar "pcre-8.42.tar.gz" "pcre-8.42"
+  Set-Location pcre-8.42\pcre-8.42
   RunConfigure "--enable-utf --enable-unicode-properties --prefix=$Env:MINGW_BASE_DIR_BASH"
```

Comment:
Can I ask if `--enable-pcre16` can be enabled here and now? I think the *nix packages tend to and I *will* **need** it in the near future to get support for UTF-16 (a.k.a. Qt `QString`s), if we get it working now whilst you are tweaking the 'Doze build environment it will be one less thing to fix-up later... 🙏

Reply:
Done, though we should think about using QRegularExpression instead of the PCRE library directly. QRegularExpression uses PCRE under the hood (statically compiled in if I read the comments correctly).

That way we'd have one library less to ship and all Qt needs are automatically taken care of. The lua regex library (which is the other part that depends on the PCRE lib) also statically compiles the pcre library in.

Reply:
Ignore the above. Apparently, I read something wrong about the `-` flag of `ld`.

Reply:
We are currently storing/processing the regular expressions as UTF-8 I think because that is the *quantum* for strings as the Lua system that the user interacts with - the reason I'm asking for UTF-16 as well is to work with the Qt C++ side of things which is what the C++ application core works with (all the Qt display widgets) - if there is a straightforward way that the two could be combined I'd like to know how - as I have that significant unpublished PR that needs the UTF-16 support to enable me to get triggering to behave correctly with non-ASCII and possibly non-BMP characters and I do not see a way around running the pcre functions twice - the second time using pcre16 analogues but operating on `QString`s compared to the others on `char *` filled with UTF-8 encoded text... 🤷‍♂️